### PR TITLE
Adapting to updated vertex selection in converter

### PR DESCRIPTION
**Pull request by jgrosseo**

### CODE REVIEW

File: Analysis/Tasks/PWGCF/correlations.cxx [hunk 1]
```diff
@@ -163,7 +164,11 @@ struct CorrelationTask {
 
     same->fillEvent(centrality, CorrelationContainer::kCFStepTriggered);
 
-    // vertex already checked as filter
+    // vertex range already checked as filter, but bitwise operations not yet supported
+    if (collision.flags() != 0 && collision.flags() & aod::collision::CollisionFlagsRun2::Run2VertexerTracks != aod::collision::CollisionFlagsRun2::Run2VertexerTracks) {
+      return;
+    }
```
Comment (iarsene) [inline Analysis/Tasks/PWGCF/correlations.cxx]: If i understand correctly, here you check that the flags has checked ONLY the Run2VertexerTracks bit. I guess the conditions are evaluated correctly, but maybe you can add some parantheses to make it more readable.

**Review by iarsene — DISMISSED** [review]

**Comment by iarsene** [comment]
@jgrosseo The macOS error is real. Some more parantheses are needed around the expressions i commented above.

**Comment by jgrosseo** [comment]
@iarsene I did follow your suggestion, see here, it says "outdated" next to your comment:
image
But apparently, I put them in the wrong place... :-( Well, it is hard to test, it does not create a warning on linux...

**Comment by iarsene** [comment]
@jgrosseo I have the same problem :)

**Pull request merged**